syntax = "proto3";
package antikeimena;

message Sensor{
    uint32 odometry_left   =  1;
    uint32 odometry_right  =  2;
    float  battery_voltage =  3;
    float  temperature     =  4;
    uint32 gps_fix         =  5;
    float  gps_satellites  =  6;
    uint32 gps_year        =  7;
    uint32 gps_month       =  8;
    uint32 gps_day         =  9;
    float  gps_altitude    = 10;
    float  gps_latitude    = 11;
    float  gps_longitude   = 12;
    float  gps_speed       = 13;
    float  gps_angle       = 14;
    uint32 ultrasonic_01   = 15;
    uint32 ultrasonic_02   = 16;
    uint32 ultrasonic_03   = 17;
    uint32 ultrasonic_04   = 18;
    uint32 ultrasonic_05   = 19;
    uint32 ultrasonic_06   = 20;
    uint32 ultrasonic_07   = 21;
    uint32 ultrasonic_08   = 22;
    uint32 ultrasonic_09   = 23;
    uint32 ultrasonic_10   = 24;
}